bindata/amq_interconnect/002-interconnect.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: interconnectedcloud.github.io/v1alpha1
+kind: Interconnect
+metadata:
+  name: amq-interconnect
+  labels: {}
+  namespace: {{ .Namespace }}
+spec:
+  users: interconnect-secret
+  deploymentPlan:
+    placement: Any
+    role: interior
+    size: 2
+    #size: {{ .InterconnectReplicas }}
```

Comment:
need to look into this as it was failing to create it due to a type conversion error